.circleci/config.yml

```diff
@@ -339,6 +337,17 @@ jobs:
       - run: yarn workspaces foreach --all --topological-dev --verbose run build:ts
       - run: yarn workspaces foreach --all --verbose run size-contracts
 
+  verify-storage:
+    docker:
+      - image: cimg/node:<< pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - install-foundry
+      - yarn-install
+      - run: yarn workspaces foreach --all --topological-dev --verbose run build:ts
+      - run: yarn workspaces foreach --all --topological-dev --verbose run build:contracts
```

Comment:
Turns out we did not run storage dump generation and storage verification in CI.